waspc/src/Wasp/Analyzer/AnalyzeError.hs

```diff
@@ -24,4 +24,4 @@ getErrorMessageAndCtx :: AnalyzeError -> (String, Ctx)
 getErrorMessageAndCtx = \case
   ParseError e -> first (("Parse error:\n" ++) . indent 2) $ PE.getErrorMessageAndCtx e
   TypeError e -> first (("Type error:\n" ++) . indent 2) $ TE.getErrorMessageAndCtx e
-  EvaluationError _e -> error "TODO"
+  EvaluationError e -> first (("Evaluation error:\n" ++) . indent 2) $ EE.getErrorMessageAndCtx e
```

Comment:
How informative is it to the user to know that something is an "evaluation" error? Parse and Type error sound more intuitive (especially Type), but the evaluation is probably the least informative, right? I guess there is no other word that would make things clearer?

Reply:
Yeah, hard to say, but as you said, we don't have a better name right now. We could make it more general, just "Compiler error", but I am not sure if that brings any value also.